src/open_inwoner/accounts/middleware.py

```diff
@@ -29,18 +29,23 @@ def __call__(self, request):
             # If the user is currently not editing their information, but it is required
             # redirect to that view.
 
-            # DigiD can be disabled, in which case the digid app isn't available
-            digid_logout = "/digid/logout/"
             try:
                 digid_logout = reverse("digid:logout")
-            except:  # nosec
-                pass
-            if (
-                not request.path.startswith(
-                    (necessary_fields_url, reverse("logout"), digid_logout)
-                )
-                and request.user.require_necessary_fields()
-            ):
-                return HttpResponseRedirect(necessary_fields_url)
+                digid_slo_redirect = reverse("slo-redirect")
+            except NoReverseMatch:
+                return self.get_response(request)
```

Comment:
This could have been a `pass`, since there is a fall-through at the end of the method after the try/except/else.

But it is not a blocking matter so I approve the PR, and @pi-sigma can mark this as ready.